docs/background.md

```diff
@@ -0,0 +1,12 @@
+# What is MOLGENIS?
```

Comment:
Duplicates README.md, is this necessary?

Reply:
For gitbook it is. We could leave the background.md out of this PR for now?

Reply:
It is used in vuepress.

Reply:
I deleted it.